src/CommunityToolkit/Xamarin.CommunityToolkit/Behaviors/EventToCommandBehavior.shared.cs

```diff
@@ -101,9 +101,10 @@ void RegisterEvent()
 			if (View == null || string.IsNullOrWhiteSpace(eventName))
 				return;
 
-			eventInfo = View.GetType().GetRuntimeEvent(eventName) ??
+			eventInfo = View.GetType()?.GetRuntimeEvent(eventName) ??
```

Comment:
The type here can't be null, we checked on line 101

Reply:
This null check is for the return value of `GetType()`.

In newer versions of the Roslyn analyzers, `GetType()` returns a nullable, `Type?`. The CI build server, which is using MSBuild 16.9, kept yelling at me until I added `GetType()?.`.

Reply:
On Windows, I found that this command will use the latest analyzers. 

So, even though Visual Studio doesn't show a Nullable error, this MSBuild command will:

```bash
"C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\MSBuild" Xamarin.CommunityToolkit.sln /restore /t:Build /p:ContinuousIntegrationBuild=true /p:Deterministic=false /p:configuration="Release"
```